lib/site_prism/section.rb

```diff
@@ -15,12 +15,12 @@ def initialize(parent, root_element)
       Capybara.within(@root_element) { yield(self) } if block_given?
     end
 
-    def visible?
-      root_element.visible?
+    def page
+      @root_element || Capybara.current_session
     end
 
-    def text
-      root_element.text
```

Comment:
`#text` method removed, because `Capybara::DSL_METHODS` includes `:text`:

https://github.com/teamcapybara/capybara/blob/30d037a0050085be996084bbf6538c3571bedea4/lib/capybara/session.rb#L34
https://github.com/teamcapybara/capybara/blob/master/lib/capybara/dsl.rb#L50